app/routes.js

```diff
@@ -1414,3 +909,7 @@ router.get('/prototype-admin/show-data', (req, res, next) => {
   next()
 })
 
+/* Use the routes file in pages for adding and editing questions */
+router.use('/pages', require('./views/form-designer/pages/\_routes'))
+
+module.exports = router
```

Comment:
This line shouldn't be necessary any more 🤔 

```suggestion
```

Reply:
Thanks for taking a look @lfdebrux... This is the only line I seem to be able to remove without it falling over